src/tests.rs

```diff
@@ -177,14 +229,14 @@ pub mod serverlesstest {
                     running: AtomicBool::new(true),
                     runtime_path: "./runtime/".to_owned(),
                 }))
-                .configure(handler::config),
+                .default_service(web::to(handler::serverless)),
```

Comment:
This test is for invalid payload for the corresponding TxHash, but that should not be part of this code. Also, it is returning `StatusCode::Ok`, so do we even need this test?